### ISSUE TITLE
room function related error

### ISSUE THREAD

**Issue by kis87988** [botorange/wechaty-puppet-padpro#17]
```log
22:34:16 VERB Room topic()
22:34:44 SILL PadproManager throttleQueue emit heartbeat.
22:34:44 SILL PuppetPadpro startWatchdog() padproManager.on(heartbeat)
22:34:44 SILL Wechaty memoryCheck() free: 372 MB, require: 4 MB
22:34:44 SILL PadproManager syncMessage()
22:34:44 SILL DedupeApi dedupe() no need to dedupe api GrpcSyncMessage.
22:34:45 SILL WechatGateway sendLong() reqSeq: 503479055
22:34:45 SILL WechatGateway sendLong() receive back package size: 246
22:34:45 SILL GrpcGateway parse(GrpcSyncMessage) get response: {"data":null}
22:34:45 VERB PadproManager syncMessage() got 0 message(s).
22:34:52 ERR WechatGateway Error happened when call api: GrpcGetContact, params: {"UserNameList":"123@chatroom"}
{ Error: connect ETIMEDOUT 203.205.129.101:80
    at TCPConnectWrap.afterConnect [as oncomplete] (net.js:1113:14)
  errno: [32m'ETIMEDOUT'[39m,
  code: [32m'ETIMEDOUT'[39m,
  syscall: [32m'connect'[39m,
  address: [32m'203.205.129.101'[39m,
  port: [33m80[39m }
22:34:52 SILL DedupeApi dedupeApi(GrpcGetContact) got results from external service, resolve 0 duplicate api calls.
22:34:52 ERR ready event fail in found room manager 
22:34:52 VERB Wechaty say(Time: 12/06/2018, 22:34:52
ready event fail in found room manager)
```

**Comment by windmemory** [comment]
Should be fixed in `0.0.25`, please test with it.

**Comment by kis87988** [comment]
Not happening now, so I am going to close the issue
